its/src/test/java/com/sonar/it/scanner/msbuild/sonarqube/CppTest.java

```diff
@@ -50,18 +51,19 @@ class CppTest {
 
   @Test
   void testCppOnly() throws Exception {
-    String projectKey = "cpp";
-    String fileKey = "cpp:ConsoleApp/ConsoleApp.cpp";
+    var context = AnalysisContext.forServer("CppSolution");
+    String projectKey = context.projectKey;
+    String fileKey = projectKey + ":ConsoleApp/ConsoleApp.cpp";
 
-    ORCHESTRATOR.getServer().provisionProject(projectKey, "Cpp");
+    ORCHESTRATOR.getServer().provisionProject(projectKey, projectKey);
```

Comment:
Just FYI, no changes here (yet, unless there will be a rebase conflict)

I already have a PR in progress to have `context.setQualityProfile(QualityProfile.CPP_S106);`